ciao-sdk/tenant.go

```diff
@@ -0,0 +1,156 @@
+package sdk
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/ciao-project/ciao/client"
+	"github.com/intel/tfortools"
+	"github.com/pkg/errors"
+)
+
+func GetTenants(c *client.Client, flags CommandOpts) (bytes.Buffer, error) {
+	var t *template.Template
+	var result bytes.Buffer
+
+	if c.Template != "" {
+		var err error
```

Comment:
You could factorize some of the code here.  The only difference I see between the two blocks is the template program.